.gitignore

```diff
@@ -10,6 +10,7 @@
 *.dll
 *.so
 *.dylib
+bin/
```

Comment:
Will you move this new rule to the bottom like this?

```
## Custom Rules

bin/
```

We keep these `## X.gitignore` sections in-sync with the [community-maintained files here](https://github.com/github/gitignore).